mtags/src/main/scala/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -179,7 +179,7 @@ case class ScalaPresentationCompiler(
     val (isSuccess, unprocessed) =
       settings.processArguments(options, processAll = true)
     require(isSuccess, unprocessed)
-    require(unprocessed.isEmpty, unprocessed)
+    logger.warning(s"Unknown compiler options: ${unprocessed.mkString(", ")}")
```

Comment:
```suggestion
if (unprocessed.nonEmpty) {
  logger.warning(s"Unknown compiler options: ${unprocessed.mkString(", ")}")
}
```

Reply:
omg! Thanks, I haven't noticed it. :fearful: